ydb/core/testlib/basics/services.cpp

```diff
@@ -320,7 +320,7 @@ namespace NPDisk {
     namespace {
 
         void AddReplicas(TStateStorageInfo::TRingGroup& group, const TVector<TActorId>& replicas) {
-            group.NToSelect = replicas.size();
+            group.NToSelect = group.NToSelect ?: replicas.size();
```

Comment:
Asked AI about this operator `?:` (it seems non-standard).

Important notes:
- This is not standard C++ - it's a GNU compiler extension
- It will only compile if you're using GCC or Clang (which supports many GNU extensions)
- MSVC typically does not support this extension
- To ensure portability, you should use the full ternary operator syntax

I believe we should change it to the standard ternary operator like this:

```suggestion
            group.NToSelect = group.NToSelect ? group.NToSelect : replicas.size();
```

Reply:
Thank you, I have made the correction in this PR: https://github.com/ydb-platform/ydb/pull/21190